.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Delete this entire file and references to Husky if it's not being used 🤖

Reply:
@John-Paul-Larkin

Reply:
Oh yeah. 
I didn't delete it because I thought there may be other uses for it?
I actually thought we might use it to run prettier on pre commit.

Reply:
I don't see anywhere else we need it so get rid of it everywhere I guess.